src/pages/Notifications/Overview/CustomDataListItem.tsx

```diff
@@ -36,7 +36,7 @@ interface CustomDataListItemProps {
   heading: string;
   linkTitle?: string;
   linkTarget?: string;
-  expandableContent: string;
+  expandableContent: ReactNode;
```

Comment:
This should be any, I see it is string in some places

Reply:
Typescript here won't allow `any`, it gets very angry. It was yelling at me about it being an Element where I added the `<span>` to be able to add the link in the `expandableContent`.